tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -364,6 +364,40 @@ function test_sync_post_includes_dont_email_post_to_subs() {
 		$this->assertEquals( true, $post_on_server->dont_email_post_to_subs );
 	}
 
+
+	function test_sync_publicize_works_as_expected() {
```

Comment:
I do not. They both should have `$post->dont_email_post_to_subs` set to false since I would want the subscriptions to be sent out. In the case of the factory->post->create that post has a status of draft and so we would not be sending out the subscription for that anyways.